lib_i2s/api/i2s.h

```diff
@@ -232,6 +232,7 @@ void i2s_master(client i2s_callback_if i2s_i,
  *  \param num_out        The number of output data ports
  *  \param p_din          An array of data input ports
  *  \param num_in         The number of input data ports
+ *  \param num_data_bits  The number of bits per data word
```

Comment:
This is a breaking change but I can't think of a way of avoiding this. No problem - will just need to ensure a major version change https://semver.org